app/presenters/base_presenter.rb

```diff
@@ -25,4 +25,11 @@ def current_user
   #   @template.send(*args, &block)
   # end
 
+  def mentioned_users(content)
```

Comment:
Would possibly abstract this into a class of it’s own, will make it easily testable and fire some unit tests around it